CHANGELOG.md

```diff
@@ -9,8 +9,9 @@
 - On `generate csv`, populate a CSV manifest’s `spec.icon`, `spec.keywords`, and `spec.mantainers` fields with empty values to better inform users how to add data. ([#2521](https://github.com/operator-framework/operator-sdk/pull/2521))
 - Scaffold code in `cmd/manager/main.go` for Go operators and add logic to Ansible/Helm operators to handle [multinamespace caching](https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/cache#MultiNamespacedCacheBuilder) if `WATCH_NAMESPACE` contains multiple namespaces. ([#2522](https://github.com/operator-framework/operator-sdk/pull/2522))
 - Add a new flag option (`--skip-cleanup-error`) to the test framework to allow skip the function which will remove all artefacts when an error be faced to perform this operation.   ([#2512](https://github.com/operator-framework/operator-sdk/pull/2512))
-- Add event stats output to the operator logs for Ansible based-operators. ([2580](https://github.com/operator-framework/operator-sdk/pull/2580))   
-- Improve Ansible logs by allowing output the full Ansible result for Ansible based-operators configurable by environment variable. ([2589](https://github.com/operator-framework/operator-sdk/pull/2589))   
+- Add event stats output to the operator logs for Ansible based-operators. ([2580](https://github.com/operator-framework/operator-sdk/pull/2580))
+- Improve Ansible logs by allowing output the full Ansible result for Ansible based-operators configurable by environment variable. ([2589](https://github.com/operator-framework/operator-sdk/pull/2589))
+- Add a new flag option `--olm-namespace` to `operator-sdk run` and `operator-sdk olm status` command, which allows to specify the namespace in which OLM is installed
```

Comment:
How the `--olm-namespace`  will be used with `operator-sdk run`? I mean, why is required to add this  flag to `operator-sdk run`?

Reply:
the `operator-sdk run --olm` command first checks to see if OLM is installed

https://github.com/operator-framework/operator-sdk/blob/master/internal/olm/operator/manager.go#L135

the same applies for cleanup as well

https://github.com/operator-framework/operator-sdk/blob/master/internal/olm/operator/manager.go#L226

so a correct OLM namespace is required for these functions to complete successfully.

Reply:
So, wdyt about to improve it such as:

```suggestion
- Add a new flag option `--olm-namespace` to `operator-sdk run --olm` sub-commands, which allows specifying the namespace where the OLM is installed
```

Reply:
@camilamacedo86 the following makes more sense, since `--olm-namespace` is being added to both subcommands.

```suggestion
- Add a new flag option `--olm-namespace` to `operator-sdk run --olm` and `operator-sdk olm` subcommands, which allows to specify the namespace in which OLM is (to be) installed. ([#2613](https://github.com/operator-framework/operator-sdk/pull/2613))
```

Reply:
@estroz  I have added the flags only to these sucommands:

```
operator-sdk run --olm
operator-sdk olm status
operator-sdk cleanup
```
and not to these olm subcommands intentionally.

```
operator-sdk olm install
operator-sdk olm uninstall
```
Since the code downloads and uses these OLM manifests from https://github.com/operator-framework/operator-lifecycle-manager/releases/download/0.14.1/olm.yaml , they create a namespace olm and other objects have namespace set to olm already

Reply:
if we want to override the install/uninstall to a different namespace, i think it would be better to do that in a separate PR

Reply:
do we want to add them to `olm install/uninstall` subcommands as well ?

Reply:
Nah, its probably better to assume OLM manifests for a particular version know how to (un)install themselves.